prometheus_juju_exporter/config_default.yaml

```diff
@@ -3,7 +3,9 @@ customer:
   cloud_name: "example_cloud"
 
 juju:
-  controller_endpoint: "" # e.g. 192.168.1.100:17070
+  controller_endpoint: ""
```

Comment:
Just want to make a note here: I understand you are keeping the name here for backward compatibility. Though I'd be happier if we can update it to "controller_endpoints", I think this is a compromise we have to make.

Reply:
Sorry, I'm not sure I understand. Do you agree that keeping it `controller_endpoint` for backward compatibility is acceptable compromise? (Because I do :laughing:). Also, given that this option can accept single string as well as list, neither `controller_endpoint` nor `controller_endpoints` is absolutely correct.

Reply:
Yes, I agree with keeping the name :) I just wanted to make a note here for future reference or in case other reviewers have different opinions.